packages/fp-constructor/index.js

```diff
@@ -2,6 +2,10 @@ var compose = require('@stamp/compose');
 
 module.exports = compose({
   composers: [function (opts) {
+    opts.stamp.of = opts.stamp;
     opts.stamp.constructor = opts.stamp;
+    opts.stamp.compose.methods = Object.assign({}, opts.stamp.compose.methods, {
```

Comment:
There is no benefit in composers to create new (often large) object. Composers were designed to mutate stamps. Please, remove new object creation. This should suffice:
```js
opts.stamp.compose.methods.constructor = opts.stamp;
```